src/main/java/net/projectzombie/region_rotation/modules/RegionState.java

```diff
@@ -36,16 +39,16 @@
     private String rotateBroadcastMessage = null;
 
     public RegionState(final String regionName,
```

Comment:
I think you already did that yourself. You have a getRegionChests(), and it looks to work.